pulp_2to3_migration/app/migration.py

```diff
@@ -177,17 +179,19 @@
         message='Migrating distributors to Pulp 3', code='migrating.distributors', total=0
     )
     with ProgressReport(**progress_data) as pb:
-        pulp2distributors_qs = Pulp2Distributor.objects.filter(
-            pulp3_distribution=None,
-            pulp3_publication=None,
-            not_in_plan=False)
-        pb.total = pulp2distributors_qs.count()
-        pb.save()
+        for plugin in plan.get_plugin_plans():
+            pulp2distributors_qs = Pulp2Distributor.objects.filter(
+                pulp3_distribution=None,
+                pulp3_publication=None,
+                not_in_plan=False,
+                pulp2_type_id=plugin.type
```

Comment:
when i run the migration it does not create any publications or distributions, i think because of this line.
In [3]: Pulp2Distributor.objects.all()[0].pulp2_type_id                                                                                                                                                            
Out[3]: 'iso_distributor'

in plugin.type you will have 'iso'

Reply:
you can fetch the types for distributors from the plugin.migrator.distriubutor_migrators.keys()

Reply:
@dralley you applied this logic to pulp2repos and pulp2 distributors, what about pulp2importers?

Reply:
Strange, I didn't seem to have that issue -- but I'll check again.

Reply:
> Strange, I didn't seem to have that issue -- but I'll check again.

https://paste.centos.org/view/0de7e0d2